features/cache.feature

```diff
@@ -210,3 +210,123 @@ Feature: Managed the WordPress object cache
 
     When I run `wp cache supports set_multiple`
     Then the return code should be 0
+
+  Scenario: Nested values from cache can be retrieved at any depth.
+    Given a WP install
+    And a wp-content/mu-plugins/test-harness.php file:
+      """
+      <?php
+      $set_foo = function(){
+        wp_cache_set( 'my_key', ['foo' => 'bar'] );
+        wp_cache_set( 'my_key_2', ['foo' => ['bar' => 'baz']] );
+        wp_cache_set( 'my_key_3', ['foo' => 'bar_custom'], 'my_custom_group' );
+      };
+
+      WP_CLI::add_hook( 'before_invoke:cache pluck', $set_foo );
+      """
+
+    When I try `wp cache pluck my_key foo`
+    Then STDOUT should be:
+      """
+      bar
+      """
+
+    When I try `wp cache pluck my_key_2 foo bar`
+    Then STDOUT should be:
+      """
+      baz
+      """
+
+    When I try `wp cache pluck my_key_3 foo --group=my_custom_group`
+    Then STDOUT should be:
+      """
+      bar_custom
+      """
+
+  Scenario: Nested values from cache can be updated at any depth.
+    Given a WP install
+    And a wp-content/mu-plugins/test-harness.php file:
+      """
+      <?php
+      $set_foo = function(){
+        wp_cache_set( 'my_key', ['foo' => 'bar'] );
+        wp_cache_set( 'other_key', ['fuz' => 'biz'] );
+
+        $complex_key = (object) [
+            'foo' => (object) [
+                'bar' => (object) [
+                    'baz' => 2,
+                ],
+            ],
+        ];
+        wp_cache_set( 'complex_key', $complex_key );
+      };
+
+      WP_CLI::add_hook( 'before_invoke:cache patch', $set_foo );
+      """
+
+    When I try `wp cache patch insert my_key fuz baz`
```

Comment:
```suggestion
    When I run `wp cache patch insert my_key fuz baz`
```

Reply:
All of the commands that are successful should use `When I run`, not `When I try`. It looks like there are multiple occurrences.